packages/forklift-console-plugin/src/modules/Providers/views/create/components/VSphereProviderCreateForm.tsx

```diff
@@ -107,19 +110,16 @@ export const VSphereProviderCreateForm: React.FC<VSphereProviderCreateFormProps>
         label={t('URL')}
         isRequired
         fieldId="url"
-        helperText={t(
-          'Enter the URL of the SDK endpoint in vCenter. Ensure it includes the "sdk" path, e.g., https://vcenter.com/sdk.',
-        )}
+        helperText={state.validation.urlHelperText}
         validated={state.validation.url}
-        helperTextInvalid={state.validation.urlHelperTextInvalid}
+        helperTextInvalid={state.validation.urlHelperText}
       >
         <TextInput
           isRequired
           type="text"
           id="url"
           name="url"
           value={url}
-          validated={state.validation.url}
```

Comment:
why did you drop the error / warning notification ?

Patternfly suggest informing users about erros and warnings, see:
https://www.patternfly.org/components/forms/text-input#invalid

Reply:
please see https://github.com/kubev2v/forklift-console-plugin/pull/744#discussion_r1342176572

It's set here https://github.com/kubev2v/forklift-console-plugin/pull/745/files#diff-78ab8802dcf0222dfbd15eeeac4d1f58cefaec2614e5900cbd2146cd8738a9d8R114 so no need to duplicate.

Reply:
My mistake, the validation state icon within the text field was removed due to this change. I'll add it.